cli/src/config/merge_tools.toml

```diff
@@ -51,3 +51,9 @@ merge-args = ["--wait", "--merge", "$left", "$right", "$base", "$output"]
 # markers. Unfortunately, it does not seem to be able to output conflict markers when
 # the user only resolves some of the conflicts.
 merge-tool-edits-conflict-markers = true
+
+[merge-tools.vscodium]
```

Comment:
nit: I would add some comment so we'll copy new settings from vscode to vscodium.
```
# free/libre distribution of vscode, functionally is more or less the same
```

Reply:
good idea, i added a comment